wagtail/admin/userbar.py

```diff
@@ -24,75 +27,137 @@ def render(self, request):
 
 
 class AccessibilityItem(BaseItem):
+    """A userbar item that runs the accessibility checker."""
+
+    #: The template to use for rendering the item.
     template = "wagtailadmin/userbar/item_accessibility.html"
 
-    def get_axe_configuration(self):
+    #: A list of CSS selector(s) to test specific parts of the page.
+    #: For more details, see `Axe documentation <https://github.com/dequelabs/axe-core/blob/master/doc/context.md#the-include-property>`__.
+    axe_include = ["body"]
+
+    #: A list of CSS selector(s) to exclude specific parts of the page from testing.
+    #: For more details, see `Axe documentation <https://github.com/dequelabs/axe-core/blob/master/doc/context.md#exclude-elements-from-test>`__.
+    axe_exclude = []
+
+    # Make sure that the userbar is not tested.
+    _axe_default_exclude = [{"fromShadowDOM": ["wagtail-userbar"]}]
+
+    #: A list of `axe-core tags <https://github.com/dequelabs/axe-core/blob/master/doc/API.md#axe-core-tags>`_
+    #: or a list of `axe-core rule IDs <https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md>`_
+    #: (not a mix of both).
+    axe_run_only = [
+        "button-name",
+        "empty-heading",
+        "empty-table-header",
+        "frame-title",
+        "heading-order",
+        "input-button-name",
+        "link-name",
+        "p-as-heading",
+    ]
+
+    #: A dictionary that maps axe-core rule IDs to a dictionary of rule options,
+    #: commonly in the format of ``{"enabled": True/False}``. This can be used in
```

Comment:
I wrote `commonly` here but I'm pretty sure that's the only format currently accepted by Axe. I'm wondering whether we should simplify the developer-facing interface to just a dictionary that maps rule IDs to boolean values, and wrap the boolean in a `{"enabled": value}` object ourself, but at the same time I personally think it's better to be as close to Axe's config format as possible to make it transparent and not so magical to developers.

https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter

This is different from the `rules` object in `axe.configure()` that accepts more fine-grained options: https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#parameters-1

(I tried using the options like the one in `configure()` but it doesn't seem to work)

Reply:
Yes, I think it’s better to stick to how Axe defines its options so there’s less magic.